### PR TITLE
Add the enter key support in grid block to create the default text bl…

### PR DESCRIPTION
**Pull request by iFlameing**
…ock below

@sneridagh screencast

https://user-images.githubusercontent.com/33936987/132359070-d375db3d-4148-4587-ac3c-14fd5ff2b031.mp4

### CODE REVIEW

File: src/components/Grid/Edit.jsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import { reorderArray, replaceItemOfArray } from '../../helpers';
 
 import { getAllowedBlocks } from '../utils';
 import templates from './templates';
+import config from '@plone/volto/registry';
```
Comment (sneridagh) [inline src/components/Grid/Edit.jsx:26]: Why this addition?

**Review by sneridagh — APPROVED** [review]
LGTM! Make sure that the block still has the desired behaviour, I guess Cypress tests would catch any regression but just to be sure! 

Congrats! Thanks a lot!

File: src/components/Grid/Edit.jsx [hunk 2]
```diff
@@ -265,8 +268,19 @@ class EditGrid extends Component {
           // for bringing the Grid sidebar alive once you have selected an inner block
           onClick={(e) => {
             this.setState({ selectedColumnIndex: null });
+            this.container.current.focus();
           }}
+          onKeyDown={(e) => {
+            this.props.onAddBlock(
+              config.settings.defaultBlockType,
```
Comment (iFlameing) [inline src/components/Grid/Edit.jsx:275]: @sneridagh because of this line.

File: src/components/Grid/Edit.jsx (hunk 1 above)
Comment (iFlameing) [reply src/components/Grid/Edit.jsx:26]: @sneridagh see the below comment

**Comment by iFlameing** [comment]
@sneridagh first time cypress fails but it is not related to my PRl. Let's wait for a second time.

**Comment by iFlameing** [comment]
@sneridagh test is failing because of the objectbrowser  selection. 
Blocks Acceptance Tests -- Text Block Tests -- As editor I can add a Teaser Grid (failed)

**Comment by iFlameing** [comment]
@sneridagh github action is happy now

**Review by sneridagh — APPROVED** [review]

**Pull request merged**